crates/pathfinder/examples/fix_missing_classes.rs

```diff
@@ -0,0 +1,110 @@
+use anyhow::Context;
+use pathfinder_lib::{
+    core::{Chain, ClassHash, StarknetBlockNumber},
+    sequencer::reply::transaction::Type as TransactionType,
+    sequencer::{Client, ClientApi},
+    state::CompressedContract,
+    storage::{
+        ContractCodeTable, StarknetBlocksBlockId, StarknetBlocksTable, StarknetTransactionsTable,
+        Storage,
+    },
+};
+
+/// Add missing declared classes.
+///
+/// Iterates over all transactions in the database and checks if we have the classes downloaded
+/// for all declare transactions, downloading missing classes.
+///
+/// Usage:
+/// `cargo run --release -p pathfinder --example fix_missing_classes ./mainnet.sqlite`
```

Comment:
Missing the chain arg I think. Not required if we use the auto-detection of the database.